.changeset/two-crews-accept.md

```diff
@@ -0,0 +1,14 @@
+---
+'@backstage/plugin-catalog-backend-module-gitlab': minor
```

Comment:
is this a breaking change for users if not this should be a `patch` changeset as the package is below version 1.x? :)

Reply:
No breaking changes, what was working before should still be working now. Will change it to `patch` then